sdk/src/test/java/com/deploygate/sdk/DeployGateInterfaceTest.java

```diff
@@ -364,4 +364,14 @@ public void composeComment__String() {
     public void getDistributionUserName() {
         Truth.assertThat(DeployGate.getDistributionUserName()).isNull();
     }
+
+    @Test
+    public void getBuildEnvironment() {
+        DeployGate.getBuildEnvironment();
```

Comment:
Please add *assertions* to testcases of functions that return values. Calling this function does not definitely mean that they never return null.